jts/pom.xml

```diff
@@ -18,21 +18,33 @@
 	
 	<groupId>org.codice.thirdparty</groupId>
 	<artifactId>jts</artifactId>
-	<version>1.14.0_1</version>
```

Comment:
❗️ The dependencies being exported at the bottom should all be exported as 1.16.0, not 1.14.0